app/src/main/java/com/kickstarter/mock/services/MockApolloClientV2.kt

```diff
@@ -147,6 +147,9 @@ open class MockApolloClientV2 : ApolloClientTypeV2 {
         return io.reactivex.Observable.just("")
     }
 
+    override suspend fun _getStoredCards(): List<StoredCard> =
+        getStoredCards().blockingSingle() // or `blockingFirst()`
```

Comment:
blocking a thread worries me a bit, we need to be sure then to add to proper dispatchers. It is a common scenario to spam several calls within the same VM